index.js

```diff
@@ -23,18 +23,19 @@ export type ResolveOptions = {
 */
 
 function getImportSources(filePath, parserOpts) {
-  let file = loadFileSync(filePath, parserOpts);
+  const fileReg = /^(?!\.(gif|jpg|png|ico|less|css|svg)$).*\.(gif|jpg|png|ico|less|css|svg)$/;
```

Comment:
Easier to match files we do care about rather than the files that we do not. It'd probably be better to add a new option:

```js
collectImportsSync(entry, parserOpts, resolveOpts, { extensions: ['js', 'jsx', 'babel'] })
```

We should default to caring about `js`, `jsx`, and `babel`, since those are Babel's defaults.